Gems/LyShine/Code/Source/RenderGraph.cpp

```diff
@@ -148,16 +157,13 @@ namespace LyShine
         drawSrg->SetConstant(uiShaderData.m_viewProjInputIndex, modelViewProjMat);
 
         drawSrg->Compile();
-
+        
         // Add the indexed primitives to the dynamic draw context for drawing
         //
         // [LYSHINE_ATOM_TODO][ATOM-15073] Combine into a single DrawIndexed call to take advantage of the draw call
         // optimization done by this RenderGraph. This option will be added to DynamicDrawContext. For
```

Comment:
Does this comment need to be updated?

Reply:
Yeah, I updated it below to say that we are combining it during AddPrimitive, but the previous sentiment still stands in that DynamicDrawContext ought to take care of it instead.

Reply:
The comment is a bit confusing to me as we have merged the draw calls already. It's probably not done in a memory optimized manner. Thoughts on updating the comment like so -> 

// Add the indexed primitives to the dynamic draw context for drawing
// TODO (GHI 17444): Vertex data for primitives is currently merged within AddPrimitive and then passed to 
// DynamicDrawContext. This can probably be further optimized whereby we dont waste extra memory and // provide the primitives directly to DynamicDrawContext to be added to its Ring buffer memory.

Reply:
Works for me!